README.md

```diff
@@ -87,16 +88,16 @@ A more comprehensive description of these items and their requirements can be fo
 
 ### Technical Prerequisites
 
-This project supports the following options for running:
+This project supports the following options for running the scripts in this repository to automate cluster provisioning:
 
 - Running the bash `oss-k8s.sh` script on your local machine
-- Using a Docker container to run the `oss-k8s.sh` script (Docker is required)
+- Using a Docker container to run the `oss-k8s.sh` script
 
    For more information, see [Docker Usage](./docs/user/DockerUsage.md). Using Docker to run the Terraform and Ansible scripts is recommended.
 
 #### Script Requirements
 
-The following software is required to run the SAS Viya IaC tooling here on your local system:
+The following software is required in order to run the SAS Viya IaC tools here on your local system:
 
 - [Terraform](https://www.terraform.io/downloads) - v.1.2.0
 - [Ansible](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html) - v2.12.5
```

Comment:
Versions are off and the doc seems to be missing what's currently in staging. Here's the link: https://github.com/sassoftware/viya4-iac-k8s/blob/staging/README.md#script-requirements

Reply:
Still seeing version and items not present.